server/services/taskService.ts

```diff
@@ -1,15 +1,27 @@
-import { PlacementApplicationTask, PlacementRequestTask, Reallocation, Task, TaskWrapper } from '@approved-premises/api'
+import {
+  PlacementApplicationTask,
+  PlacementRequestTask,
+  Reallocation,
+  SortDirection,
+  Task,
+  TaskWrapper,
+} from '@approved-premises/api'
 import { GroupedMatchTasks, PaginatedResponse } from '@approved-premises/ui'
 import { RestClientBuilder } from '../data'
 import TaskClient from '../data/taskClient'
 
 export default class TaskService {
   constructor(private readonly taskClientFactory: RestClientBuilder<TaskClient>) {}
 
-  async getAllReallocatable(token: string): Promise<Array<Task>> {
+  async getAllReallocatable(
+    token: string,
+    allocatedFilter: string = 'allocated' || 'unallocated',
```

Comment:
I think `allocatedFilter: 'allocated' | 'unallocated' = 'allocated'` would be better so that consumers can't pass an invalid string